tests/Api.IntegrationTests/Endpoints/SignIn/SignInTests.cs

```diff
@@ -211,4 +212,42 @@ public async Task I_receive_a_sign_in_token_for_a_valid_user_id()
         var verifySignInResponse = await client.PostAsJsonAsync("/signin/verify", new SignInVerifyDTO { Token = generateToken.Token, Origin = PasswordlessApi.OriginUrl, RPID = PasswordlessApi.RpId });
         verifySignInResponse.StatusCode.Should().Be(HttpStatusCode.OK);
     }
+
+    [Fact]
+    public async Task I_receive_an_api_exception_when_using_an_expired_token()
+    {
+        // Arrange
+        const int timeToLive = 120;
+        await using var api = await apiFixture.CreateApiAsync(testOutput);
+        using var httpClient = api.CreateClient().AddPublicKey().AddSecretKey().AddUserAgent();
+
+        using var client = api.CreateClient().AddManagementKey();
+        using var createApplicationMessage = await client.CreateApplicationAsync();
+        var userId = $"user{Guid.NewGuid():N}";
+        var accountKeysCreation = await createApplicationMessage.Content.ReadFromJsonAsync<CreateAppResultDto>();
+        client.AddPublicKey(accountKeysCreation!.ApiKey1)
+            .AddSecretKey(accountKeysCreation.ApiSecret1)
+            .AddUserAgent();
+        using var signInGenerateTokenResponse = await client.PostAsJsonAsync("signin/generate-token", new SigninTokenRequest(userId, timeToLive)
+        {
+            Origin = PasswordlessApi.OriginUrl,
+            RPID = PasswordlessApi.RpId
+        });
+        var generateToken = await signInGenerateTokenResponse.Content.ReadFromJsonAsync<SigninEndpoints.SigninTokenResponse>();
+        generateToken.Should().NotBeNull();
+        generateToken!.Token.Should().StartWith("verify_");
+
+        api.Time.Advance(TimeSpan.FromSeconds(timeToLive + 10));
+
+        // Act
+        var verifySignInResponse = await client.PostAsJsonAsync("/signin/verify", new SignInVerifyDTO { Token = generateToken.Token, Origin = PasswordlessApi.OriginUrl, RPID = PasswordlessApi.RpId });
+
+        // Assert
+        verifySignInResponse.StatusCode.Should().Be(HttpStatusCode.Forbidden);
+        var problemDetails = await verifySignInResponse.Content.ReadFromJsonAsync<ProblemDetails>();
+        problemDetails.Should().NotBeNull();
+        problemDetails!.Extensions["errorCode"]!.ToString().Should().Be("expired_token");
+        problemDetails.Status.Should().Be(403);
+        problemDetails.Title.Should().Be("The token expired 00:00:10 ago.");
```

Comment:
I know this is not part of this PR, but just thinking out loud (no need to change this in this PR or at all):

`00:00:10 ago` is not very natural to read. Maybe we should normalize the value to seconds/minutes? Could either use [Humanizer](https://github.com/Humanizr/Humanizer) or just write a simple extension method.

Reply:
We already have it in the project so I added it.